server/conn.go

```diff
@@ -133,6 +135,11 @@ func (cc *clientConn) Close() error {
 func (cc *clientConn) writeInitialHandshake() error {
 	data := make([]byte, 4, 128)
 
+	var serverCapability = defaultCapability
```

Comment:
That's a good idea. Shall we move "defaultCapability" to something like "server.Capability"?